pkg/collector/upgrade/v0_61_0-invalid.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
There is a convention in go to put files like these into `testdata` directory in the same package